.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: ruby
+
+env:
+  - TRAVIS_NODE_VERSION="6.3.0"
```

Comment:
Sorry, I forgot to comment on this earlier. This configuration is using nvm and not travis-ci convention found in the component repos. Therefore we need to target a specific version of node.